nomad/vault.go

```diff
@@ -1435,6 +1476,8 @@ func (v *vaultClient) EmitStats(period time.Duration, stopCh <-chan struct{}) {
 			stats := v.stats()
 			metrics.SetGauge([]string{"nomad", "vault", "distributed_tokens_revoking"}, float32(stats.TrackedForRevoke))
 			metrics.SetGauge([]string{"nomad", "vault", "token_ttl"}, float32(stats.TokenTTL/time.Millisecond))
+			metrics.SetGauge([]string{"nomad", "vault", "last_renewal"}, float32(stats.TimeFromLastRenewal/time.Millisecond))
+			metrics.SetGauge([]string{"nomad", "vault", "next_renewal"}, float32(stats.TimeToNextRenewal/time.Millisecond))
```

Comment:
Since these are durations potentially on the order of days, is ns a useful grain? On reflection, given metric collection intervals, is a grain longer than a second worthwhile?

Reply:
I would keep it the same unit as other metrics for consistency.

Reply:
just to update this convo, our docs needs to be updated to reflect that they all use `ms` not `ns`

Reply:
updated metric names in a separate commit